universal-application-tool-0.0.1/app/services/cloud/azure/BlobStorage.java

```diff
@@ -143,14 +143,14 @@ public String getBlobUrl(String fileName) {
     private UserDelegationKey userDelegationKey;
     private ZoneId zoneId;
 
-    AzureBlobClient(Config config) {
+    AzureBlobClient(ZoneId zoneId) {
```

Comment:
passing in the zonezId here instead of the config since we only want the zone from the config